src/sapling/sapling_operation.h

```diff
@@ -107,6 +105,13 @@ class SaplingOperation {
     CTransactionRef getFinalTxRef() { return finalTx; }
 
 private:
+    /*
+     * Cannot be nullptr. A pointer to the wallet, used to retrieve the inputs to spend, the keys to create the outputs,
+     * sapling notes and nullifiers, as well as to commit transactions.
+     * The same keystore is passed to the transaction builder in order to produce the required signatures.
+     */
+    CWallet* wallet;
```

Comment:
better to always initialize the pointers.

Reply:
added